### PR TITLE
update provisioner/attacher statefulsets

### PR DESCRIPTION
**Pull request by rootfs**
- use k8scsi image
- create plugin directory if not exist

### CODE REVIEW

**Comment by rootfs** [comment]
@sbezverk

File: deploy/kubernetes/csi-attacher.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccount: csi-attacher
       containers:
         - name: csi-attacher
-          image: csi_images/csi-attacher:latest
+          image: docker.io/k8scsi/csi-attacher:latest
```
Comment (sbezverk) [inline deploy/kubernetes/csi-attacher.yaml]: Does it have to be k8scsi? It sounds more like k8 scsi :) maybe k8s-csi?

Comment (rootfs) [reply deploy/kubernetes/csi-attacher.yaml]: this is the repo registered by kubernetes-csi team :)

Comment (sbezverk) [reply deploy/kubernetes/csi-attacher.yaml]: Ok, sounds good

Comment (dmick) [reply deploy/kubernetes/csi-attacher.yaml]: The commentary explains the change is being made, but not why; could you add a "why" for other readers, @rootfs ?

Comment (rootfs) [reply deploy/kubernetes/csi-attacher.yaml]: the image comes from [here](https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/statefulset.yaml#L75)

Comment (dmick) [reply deploy/kubernetes/csi-attacher.yaml]: ...and that's fine, but there's still no explanation of why the image location changed from csi_images/csi-attacher:latest

Comment (rootfs) [reply deploy/kubernetes/csi-attacher.yaml]: Right, thats a good point. `csi_images` is not at any public image registry, it is a local image used for development. For testing and production, the images have be hosted in a registry.

Comment (dmick) [reply deploy/kubernetes/csi-attacher.yaml]: thanks.

**Pull request merged**